lib/utils/validator-extras.js

```diff
@@ -39,7 +39,7 @@ const extensions = {
     return !this.regex(str, pattern, modifiers);
   },
   isDecimal(str) {
-    return str !== '' && !!str.match(/^(?:-?(?:[0-9]+))?(?:\.[0-9]*)?(?:[eE][+-]?(?:[0-9]+))?$/);
```

Comment:
Uhm. Aren't we calling the exact same function here and create an endless loop?

Reply:
Could it be that you wanted to remove the function altogether?

Reply:
> Could it be that you wanted to remove the function altogether?

Hello, I'm still working on this PR sorry for late response.

Reply:
I will close this issue for now and will lookup after all new TODOS soon.